examples/liquid.ini

```diff
@@ -72,6 +72,10 @@ wait_green_count = 10
 [job:zipkin]
 template = templates/zipkin.nomad
 
+# Run a custom local job, see litterbox.py
```

Comment:
For non-devs it would be easier if the job was defined in ini. E.g.:
```[job:Cat]
template = ...
```

Reply:
that still works https://github.com/liquidinvestigations/node/pull/110/files#diff-bacec7d196e27ee42066f951fd810473R130